CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### Features
 
+- `[jest-mock]` Add `contexts` member to mocks ([#12601](https://github.com/facebook/jest/pull/12601))
```

Comment:
can you sort this alphabetically?

Reply:
Sure thing, I put it with the other `[jest-mock]` entries.